codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -223,6 +228,9 @@ final void onHttp2Frame(ChannelHandlerContext ctx, Http2Frame frame) {
             if (settings.initialWindowSize() != null) {
                 initialOutboundStreamWindow = settings.initialWindowSize();
             }
+            if (propagateSettings) {
+                ctx.fireChannelRead(frame);
```

Comment:
@Scottmitch @carl-mastrangelo @ejona86 I wonder if we should just do this always and not need a special config for it... /cc @shorea

Reply:
That'd be fair, and would be fine for Http2GoAwayFrame as well (so the idea generalizes).

Reply:
Sounds good to me... what do the others think ?

Reply:
That sounds good to me, I'll make that change.